src/Console/Command/ListFixersCommand.php

```diff
@@ -0,0 +1,551 @@
+        $this->hideConfigured = $input->getOption(self::OPT_HIDE_CONFIGURED);
+        $this->hideEnabled = $input->getOption(self::OPT_HIDE_ENABLED);
+        $this->hideRisky = $input->getOption(self::OPT_HIDE_RISKY);
+        $this->hideInherited = $input->getOption(self::OPT_HIDE_INHERITED);
+        $this->hideDeprecated = $input->getOption(self::OPT_HIDE_DEPRECATED);
+        $this->hideCustom = $input->getOption(self::OPT_HIDE_CUSTOM);
+
+        if ($this->onlyConfigured && $this->hideConfigured) {
+            throw new \InvalidArgumentException(sprintf('The options "--%s" and "--%s" cannot be used together.', self::OPT_ONLY_CONFIGURED, self::OPT_HIDE_CONFIGURED));
+        }
+    }
+
+    private function createResolver(InputInterface $input): void {
+        $resolver = new ConfigurationResolver(
+            new Config(),
+            [
+                'config' => $input->getOption('config'),
+            ],
+            getcwd(),
+            $this->toolInfo
+        );
+
+        $this->configName = $resolver->getConfig()->getName();
+        $this->configFile = $resolver->getConfigFile();
+
+        $this->configurationResolver = $resolver;
+    }
+
+    private function processRuleSet(string $name, RuleSetInterface $set): void
+    {
+        /** @var bool $value */
+        foreach ($set->getRules() as $rule => $value) {
+            $this->fixerList[$rule]['in_set'][] = $name;
+            $this->fixerList[$rule]['is_inherited'] = true;
+        }
+    }
+
+    private function processFixer(FixerInterface $fixer): void
+    {
+        $this->fixerList[$fixer->getName()]['name'] = $fixer->getName();
+        $this->fixerList[$fixer->getName()]['source'] = $this->configurationResolver->getRulesSource()[$fixer->getName()] ?? null;
+        $this->fixerList[$fixer->getName()]['is_configured'] = $this->isFixerConfigured($fixer);
+        $this->fixerList[$fixer->getName()]['is_enabled'] = $this->isFixerEnabled($fixer);
+        $this->fixerList[$fixer->getName()]['is_enabled_through_inheritance'] = $this->isFixerEnabledThroughInheritance($fixer);
+        $this->fixerList[$fixer->getName()]['is_risky'] = $this->isFixerRisky($fixer);
+        $this->fixerList[$fixer->getName()]['is_inherited'] = false;
+        $this->fixerList[$fixer->getName()]['is_deprecated'] = $this->isFixerDeprecated($fixer);
+        $this->fixerList[$fixer->getName()]['is_custom'] = $this->isCustomFixer($fixer);
+    }
+
+    private function buildTable(OutputInterface $output): Table
+    {
+        $table = new Table($output);
+
+        $rows = $this->filterFixers();
+        $table->setRows($rows);
+
+        $columns = [
+            sprintf('Fixer (%d)', \count($rows)),
+        ];
+
+        if (!$this->hideRisky) {
+            $columns[] = sprintf('Risky (%d)', $this->countRiskyFixers);
+        }
+
+        if (!$this->hideInherited) {
+            $columns[] = sprintf('From set (%d)', $this->countInheritedFixers);
+        }
```

Comment:
Can we please move this column to be 2nd one, right after rules' names? I believe it's more important information than riskiness and should be next to names.